test/performance/tests/test_util/harness/lxd.py

```diff
@@ -155,13 +156,23 @@ def pull_file(self, instance_id: str, source: str, destination: str):
         except subprocess.CalledProcessError as e:
             raise HarnessError("lxc file push command failed") from e
 
-    def exec(self, instance_id: str, command: list, **kwargs):
+    def exec(self, instance_id: str, command: list, background: bool = False, **kwargs):
         if instance_id not in self.instances:
             raise HarnessError(f"unknown instance {instance_id}")
 
         LOG.debug("Execute command %s in instance %s", command, instance_id)
+
+        if ">" in " ".join(command):
+            command_str = " ".join(command)
```

Comment:
hang on... are you looking for a shell redirect here by searching for `>`.  This feels fragile.  I wonder if i can help think of a better way

Reply:
The shlex.join doesn't work with special characters so if my command has a ">" in it I am doing a regular join.